hicommonwealth/commonwealth
# User-Onboarding: Fix email assignment on SSO accounts

## Link to Issue
Closes: https://github.com/hicommonwealth/commonwealth/issues/6752

## Description of Changes
We were not storing emails from SSO accounts due to a recent change in https://github.com/hicommonwealth/commonwealth/pull/6886/files, this PR adds some of the "storing email from SSO" logic back.

## "How We Fixed It"
N/A

## Test Plan
- Create a new SSO account
- Login
- Verify that `/status` API response contains `email` set as the email granted from the SSO service used to login

## Deployment Plan
N/A

## Other Considerations
N/A
